src/identity/components/GlobalHeader/GlobalHeaderDropdown/index.tsx

```diff
@@ -126,15 +127,21 @@ export class GlobalHeaderDropdown extends React.Component<Props, State> {
               onClick={this.sendMainMenuEvent(menuItem.name)}
               key={`eventWrapper.${menuItem.name}`}
             >
-              <Dropdown.HrefItem
-                className="global-header--align-center"
+              <Dropdown.LinkItem
+                className="global-header--main-dropdown-item"
                 key={menuItem.name}
-                href={menuItem.href}
                 testID={`${this.props.mainMenuTestID}-${menuItem.name}`}
+                selected={false}
```

Comment:
`Selected` status is a mandatory element, and should be set to `false` for the dropdown to be consistent with the design here.